docs/core/preview3/tools/using-ci-with-cli.md

```diff
@@ -108,4 +163,50 @@ install:
   # .NET Core SDK binaries
   - ps: $url = "https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/$($env:CLI_VERSION)/dotnet-dev-win-x64.$($env:CLI_VERSION.ToLower()).zip"
   # follow normal installation from binaries
-```
\ No newline at end of file
+```
+
+### Visual Studio Team Services
+There are two ways of configuring the Visual Studio Team Services (VSTS) build to build .NET Core projects:
+
+1. Using the script from the manual step above and running it on your code 
+2. Creating a build out of several build steps that are configured to use .NET Core tools and are available out-of-box on VSTS 
+
+Both solutions are perfectly valid and there is no inherent flaw or advantage to any of them. That said, with the manual step you can completely control the version of the tools that you need since you download it as part of the build. On the other hand, the build is run from a script that you have to author, so that script has to take care of running the build. In this document, we will cover only the manual option, as the other one is very well documented in the overall VSTS build documentation.  
+
+> [!NOTE]
+> This document is not an exhaustive introduction into VSTS builds and assumes familiarity with how they work. 
+> If you need to brush up on basics, please visit the [VSTS build documentation](https://www.visualstudio.com/en-us/docs/build/overview). 
```

Comment:
If you need to brush up on basics, please visit -> For more information, see 

Link should not have locale